src/auth/credentials.ts

```diff
@@ -17,6 +17,6 @@
 export interface Credentials {
   refresh_token?: string;
   expiry_date?: number;
-  access_token?: string;
+  access_token?: string|null;
```

Comment:
So it can be `string` or `null` or `undefined`. Is there a way to avoid the `null`?

Reply:
The only place using null was the test, so I just changed it to undefined 🤷‍♂️